src/main/webapp/events.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>Events</title>
+  <link rel="stylesheet" href="/css/main.css">
+  <link rel="stylesheet" href="/css/events-page.css">
+  <script src="/js/events-loader.js"></script>
+  <script src="/js/navigation-loader.js"></script>
+</head>
+
+<body onload="loadNavBar();">
+  <nav>
+    <ul id="navigation">
+      <li><a href="/">Home</a></li>
+    </ul>
```

Comment:
add
<li class="dropdown">
        <a href="javascript:void(0)" class="dropbtn">More</a>
        <ul class="dropdown-content" id="dropdown-content"></ul>
      </li>
beneath the home li element so the rest of the nav bar loads